### PR TITLE
Features/remove custom dir

### PR DESCRIPTION
**Pull request by vtdave**
* Removes looking for packages out of custom directories
    * kept `packages/custom` for loading to preserve backwards compatibility
* Added `.editorconfig` to repo
* Formatted code to `.editorconfig` settings

### CODE REVIEW

File: lib/core_modules/server/ExpressEngine.js [hunk 1]
```diff
@@ -50,28 +54,28 @@ ExpressEngine.prototype.initApp = function() {
   this.app.use('/api', expressJwt({
     secret: config.secret,
     credentialsRequired: false
-  }), function(req, res, next) {
-      if (req.user) req.user = JSON.parse(decodeURI(req.user));
-      next();
+  }), function (req, res, next) {
+    if (req.user) req.user = JSON.parse(decodeURI(req.user));
```
Comment (clowenhg) [inline lib/core_modules/server/ExpressEngine.js]: Brackets for all if blocks?

**Comment by rjVapes** [comment]
Let's PR this against the releases/0.10 branch since it's a breaking change

**Comment by clowenhg** [comment]
Closing to redo pr

**Comment by timelf123** [comment]
We should probably cherry pick the change made to https://github.com/linnovate/meanio/tree/0.8 too

**Pull request closed without merging**